### PR TITLE
no printing trimmed querys in json output

### PR DESCRIPTION
**Pull request by jacobmealey**
previously if you trimmed a query and requested a json output trurl would print a set of empty key/value pairs

### CODE REVIEW

**Comment by bagder** [comment]
How about adding a test case for this as well?

File: trurl.c [hunk 1]
```diff
@@ -693,6 +693,9 @@ static void json(struct option *o, CURLU *uh)
     fputs(",\n    \"params\": [\n", stdout);
     for(i = 0 ; i < nqpairs; i++) {
       char *sep = strchr(qpairsdec[i], '=');
+      /* don't print out empty/trimmed values */
+      if(!strlen(qpairsdec[i]))
+          continue;
       if(i)
```
[the remark this answers is not in the document]
Comment (jacobmealey) [reply trurl.c:699]: Good catch! thanks

**Comment by bagder** [comment]
Ah sorry, I merged #171 which caused a merge conflict here. Can you please fix and force-push ?

**Comment by jacobmealey** [comment]
@bagder I fixed the merge conflicts, let me know if there are any other problems!

**Review by bagder — APPROVED** [review]

**Comment by bagder** [comment]
Thanks!

**Pull request merged**